### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by kernelkind**
Slider functionality works. Fixed Example image not showing up.

### CODE REVIEW

**Comment by kernelkind** [comment]
also, put app.py through black

**Comment by calvinyong** [comment]
git rm the .DS_Store's, and add that to .gitignore.

**Review by calvinyong — CHANGES_REQUESTED** [review]
Remove .DS_Store. Add .DS_Store to .gitignore

File: app.py [hunk 1]
```diff
@@ -1,107 +1,148 @@
...
-def update_output(*list_of_contents):
-    if None not in list_of_contents:
-        contents = list_of_contents[0][0]
-        children = parse_contents(contents)
+@app.callback(
+    [Output("output-image-upload", "children"), Output("bar_graph", "figure")],
+    [Input("upload-image", "contents"), Input("k-slider", "value")],
+    [State("output-image-upload", "children"), State("bar_graph", "figure")],
+)
+def update_output(uploaded_image, k_val, state_img, state_bg):
```
Comment (calvinyong) [inline app.py]: `state_bg` not used. Consider deleting the parameter if not necessary.

Comment (calvinyong) [inline app.py]: `state_bg` not used. Consider deleting the parameter if not necessary.

**Review by calvinyong — APPROVED** [review]

**Pull request merged**